uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,75 @@
+package uninstall
+
+import (
+	"time"
+	"io/ioutil"
+	"os/exec"
+	"github.com/pufferpanel/pufferd/logging"
+	"syscall"
+)
+
+
+func StartProcess(){
+	killDaemon()
+	deleteFiles()
+	deleteUser()
+}
+
+func killDaemon(){
+	exec.Command("systemctl", "pufferd", "stop").Run()
+	logging.Info("Attempting to kill all pufferd process...")
+	time.Sleep(time.Second * 5)//Giving 5 seconds to kill "correctly" all process
+	exec.Command("killall", "-9" "-u", "pufferd").Run()//"Hard killing" anything
+}
+
+func deleteUser(){
+	cmd := exec.Command("userdel", "-Z", "-r", "-f", "pufferd")
+	err := cmd.Run() //Delete pufferd and it's home dir (/var/lib/pufferd)
+
+	if err != nil{
+
+		if exitErr, ok := err.(*exec.ExitError); ok {

```

Comment:
I did not understand the point, anyway I noticed that the error were logged twice...